src/pages/STG_PanelRD.page

```diff
@@ -61,6 +61,20 @@
                     </apex:outputPanel>
                 </div>
             </div>
+            <div class="slds-form-element" style="{!IF(!isRD2Enabled, 'display:none', 'display:block')}">
+                <apex:outputLabel value="{!$ObjectType.npe03__Recurring_Donations_Settings__c.Fields.Installment_Opportunity_Auto_Creation__c.Label}" for="IOAC" styleClass="slds-form-element__label" />
+                <div class="slds-form-element__control">
+                    <div class="slds-form-element__control">
+                        <apex:outputText value="{!installmentAutoCreationLabel}" rendered="{!isReadOnlyMode}" styleClass="slds-form-element__static" />
+                        <apex:selectList value="{!stgService.stgRD.Installment_Opportunity_Auto_Creation__c}" size="1" multiselect="false" rendered="{!isEditMode}" id="IOAC" html-aria-describedby="{!$Component.IOACHelp}" styleClass="slds-select" >
```

Comment:
Ids such as `IOAC` and `IOACHelp` are not readable. Let's use more descriptive names to clarify what fields represent.